itou/metabase/management/test_populate_metabase_matomo.py

```diff
@@ -42,7 +43,97 @@
+        "\t> fetching date=2022-06-13 dashboard='tb 116 - Recrutement' segment=h",
+        "\t> fetching date=2022-06-13 dashboard='tb 129 - Analyse des publics' s",
+        "\t> fetching date=2022-06-13 dashboard='tb 136 - Prescripteurs habilité",
+        "\t> fetching date=2022-06-13 dashboard='tb 140 - ETP conventionnés' seg",
+        "\t> fetching date=2022-06-13 dashboard='tb 150 - Fiches de poste en ten",
+        "\t> fetching date=2022-06-13 dashboard='tb 32 - Acceptés en auto-prescr",
+        "\t> fetching date=2022-06-13 dashboard='tb 43 - Statistiques des emploi",
+        "\t> fetching date=2022-06-13 dashboard='tb 52 - Typologie de prescripte",
+        "\t> fetching date=2022-06-13 dashboard='tb 54 - Typologie des employeur",
+        "\t> fetching date=2022-06-13 dashboard='tb 90 - Analyse des métiers' se",
+        "> about to fetch count=10 public dashboards from Matomo.",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
+        "For more information check: https://httpstatuses.com/500",
```

Comment:
Euh DNRY ? `["For more information check: https://httpstatuses.com/500"] * 30`

Reply:
ah yeah tu as bien raison !